bgpd/rfapi/rfapi.c

```diff
@@ -383,7 +382,8 @@ void del_vnc_route(struct rfapi_descriptor *rfd,
 
 	vnc_zlog_debug_verbose(
 		"%s: peer=%p, prefix=%s, prd=%s afi=%d, safi=%d bn=%p, bn->info=%p",
-		__func__, peer, buf, buf2, afi, safi, bn,
+		__func__, peer, buf,
+		prefix_rd2str(prd, buf2, sizeof(buf2)), afi, safi, bn,
```

Comment:
This is a little ugly, why not leave rd2str in it's original place?

Reply:
because we don't run prefix_rd2str *unless* the vnc_zlog_debug_verbose actually is debugging.   And I disaggree that it's ugly.  this is the standard way it's done throughout the code base.

Reply:
Then should be consistent and do it for prefix2str/buf as well

Reply:
There is already a lot of inconsistent debug code here.  I believe these commits are a step forward and I intend to keep working on this code.  Can we just take the code as is and I'll continue working on it? 

I agree that this needs to be cleaned up and will do so.  I would just like to switch tasks for a bit and work on this as a background task for me, hence the request to get it in.